tests/integration/cri-containerd/integration-tests.sh

```diff
@@ -474,7 +474,9 @@ function main() {
 	# containerd's `cri-integration` will print the log itself.
 	trap err_report ERR
 
-	TestContainerSwap
+	# testContainerSwap is currently failing with GHA.
```

Comment:
Is it worth raising an issue to track this and linking it here?

Reply:
Let me do that right now.

Reply:
https://github.com/kata-containers/kata-containers/issues/7410

done.